README.md

```diff
@@ -7,19 +7,19 @@
 # Branches of Interest
 
 - [`0.4.x`](https://github.com/Libvisual/libvisual/tree/0.4.x) —
-  targetting version `0.4.x`,
+  targetting version 0.4.x,
   maintenance,
```

Comment:
Oops, 'targeting' is misspelled (1 t only).

Reply:
@kaixiong it's not a bug, see https://en.wiktionary.org/wiki/targetting . Now replaced with "targets" to e.g. better fit "uses".

Reply:
@hartwork, I saw that same page before commenting :laughing: Here's an interesting [link](https://english.stackexchange.com/questions/205815/is-targetted-a-standard-british-english-spelling).

Reply:
@kaixiong good link, ready to merge?

Reply:
@hartwork, yup approved.